pkg/controller/controller.go

```diff
@@ -264,8 +267,11 @@ func (c *controller) getClusterServiceClassPlanAndClusterServiceBroker(instance
 
 	servicePlan, err := c.servicePlanLister.Get(instance.Spec.ClusterServicePlanRef.Name)
 	if nil != err {
-		s := fmt.Sprintf(`ServiceInstance "%v/%v": references a non-existent ClusterServicePlan %q on ClusterServiceClass %q1`, instance.Namespace, instance.Name, instance.Spec.ExternalClusterServicePlanName, serviceClass.Spec.ExternalName)
-		glog.Warning(s)
+		s := fmt.Sprintf("References a non-existent ClusterServicePlan %q on ClusterServiceClass %q", instance.Spec.ExternalClusterServicePlanName, serviceClass.Spec.ExternalName)
```

Comment:
Can we disambiguate the k8s and external names in this PR where it isn't already done?

Reply:
sure! I can add that.